src/views/Settings/Settings.js

```diff
@@ -55,17 +58,24 @@ export default class Settings extends Component {
     try {
```

Comment:
Why do you have to try-catches in a single async function? You should only need **one**